src/RewardsStreamerMP.sol

```diff
@@ -42,50 +41,50 @@ contract RewardsStreamerMP is ReentrancyGuard {
     mapping(address account => UserInfo data) public users;
 
     constructor(address _stakingToken, address _rewardToken) {
-        STAKING_TOKEN = IERC20(_stakingToken);
+        STAKE_TOKEN = IERC20(_stakingToken);
         REWARD_TOKEN = IERC20(_rewardToken);
         lastMPUpdatedTime = block.timestamp;
     }
 
-    function stake(uint256 amount, uint256 lockPeriod) external nonReentrant {
-        if (amount == 0) {
-            revert StakingManager__AmountCannotBeZero();
+    function stake(uint256 _amount, uint256 _seconds) external onlyTrustedCodehash nonReentrant {
+        if (_amount == 0) {
+            revert StakeManager__StakeIsTooLow();
```

Comment:
what do you think about the previous `__AmountCannotBeZero`? I happy with both in the end,  but more `too low` sounds like there's a minimum while `cannot be zero` feels more like any positive amount

Reply:
The reason I am chaning to StakeIsTooLow because it would return the idea that cannot be zero and I want to keep the same error messages as possible from StakeManager.sol so we can reuse tests in both systems.

Actually, we need to verify your minimum is actually 1 token.